app-frontend/src/app/pages/projects/edit/edit.html

```diff
@@ -18,9 +18,16 @@ <h5 class="sidebar-title">Editing project</h5>
           </div>
         </li>
         <li>
-          <span class="label">Add/Remove Scenes</span>
-          <button class="btn btn-primary fixedwidth" type="button" ui-sref="projects.edit.scenes">
+          <span class="label">Scene Management</span>
+          <a class="btn btn-primary fixedwidth" ui-sref="projects.edit.scenes">
```

Comment:
We should probably have a style guide addition that describes when to use <a> and when to use <button>.  We use them pretty interchangeably.